### DISCUSSION TITLE
React native support

### DISCUSSION THREAD

**Discussion by timhaller** [dicebear/dicebear#263] [Q&A] [answered]
Hello,
I want to use the JS-library in a react native app, is it possible? I have already tried but encountered some errors with the text encoder/decoder.
I want to make a screen where the user can chose all the options and the avatar is automatically updated…
![this is the example of how i want it to look](https://user-images.githubusercontent.com/67918170/213160601-f05c7bd2-2ba6-4a0f-b9c9-2afad0255d8b.png)

**Comment by FlorianKoerner** [comment, marked as answer]
Hi @timhaller 
I published a small article on the website about React Native. I hope it will help you.

https://dicebear.com/guides/use-the-library-with-react-native

**Comment by timhaller** [comment]
Hey,
First of all sorry for writing this late 🙃

But I wanted to say thank you for the answer, it works great :)

Tim Haller
